__tests__/app.test.js

```diff
@@ -214,6 +214,72 @@ describe('/api/articles/:article_id/comments', () => {
     });
 });
 
+describe('/api/articles/:article_id', () => {
+    test('PATCH request responds with 201 with updated article votes', () => {
+        const testUpdate = { inc_votes: 1 }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(testUpdate)
+        .expect(202)
```

Comment:
Just something very minor, but might want to change your test description to fit the status code we actually use - a 202